File: Linguagem C Exercicios/Matrizes/NEGATIVOS MATRIZ.c
#include <stdio.h>

int main()
{
    int m,n;

    printf("Qual a quantidade de linhas da matriz? ");
    scanf("%d",&m);
    printf("Qual a quantidade de colunas da matriz? ");
    scanf("%d",&n);

    int mat[m][n];

    //Lendo a matriz
    for (int i=0;i<=m-1;i++){
        for (int j=0;j<=n-1;j++){
            printf("Elemento [%d,%d]: ",i,j);
            scanf("%d",&mat[i][j]);
        };
    };

    //Imprimindo os valores negativos
    printf("VALORES NEGATIVOS:\n");
    for (int i=0;i<=m-1;i++){
        for (int j=0;j<=n-1;j++){
            if (mat[i][j] < 0){
                printf("%d\n",mat[i][j]);
            };
        };
    };

    return 0;
}

File: Linguagem Java/Exercicios/NegativosMatriz.java
import java.util.Locale;
import java.util.Scanner;

public class NegativosMatriz {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int m,n;
		
		System.out.print("Qual a quantidade de linhas da matriz? ");
		m = sc.nextInt();
		System.out.print("Qual a quantidade de colunas da matriz? ");
		n = sc.nextInt();
		
		int mat[][] = new int[m][n];
		
		//Lendo a matriz
		for (int i=0;i<m;i++) {
			for (int j=0;j<n;j++) {
				System.out.print("Elemento ["+i+","+j+"]: ");
				mat[i][j] = sc.nextInt();				
			};			
		};
		
		// Numeros negativos
		System.out.println("VALORES NEGATIVOS:");
		for (int i=0;i<m;i++) {
			for (int j=0;j<n;j++) {
				if (mat[i][j] < 0) {
					System.out.println(mat[i][j]);					
				};				
			};			
		};
		
		sc.close();

	}

}
